includes/rest-api/class-sensei-rest-api-extensions-controller.php

```diff
@@ -245,7 +244,7 @@ function( $plugin ) use ( $plugin_slug ) {
 		}
 
 		$installed_plugins = array_filter(
-			$available_plugins,
+			Sensei_Extensions::instance()->get_extensions_and_woocommerce( 'plugin' ),
```

Comment:
We need to get it again to get it with the updated status (installed/activated). In the future, we probably could refactor it to just use the request status to confirm if it was properly installed or not, similar to what we do in `assets/setup-wizard/features/index.js`

In the past, it was needed because we could install/update multiple extensions at the same time.